tests/test_hasheous.py

```
@@ -0,0 +1,1559 @@
+    assert recorded[0] <= 10
+    assert recorded[1] < recorded[0], "the deadline must not reset between reads"
+
+
+def test_the_deadline_is_cleared_after_the_request():
+    with hasheous._deadline_of(10):
+        assert hasheous._deadline.at is not None
+    assert hasheous._deadline.at is None
+
+
+@pytest.mark.parametrize("mode", ["header", "body"])
+def test_a_dripping_server_cannot_pin_a_lookup(hasheous_on, monkeypatch, tls_cert, mode):
```

Ruff (0.16.1)

[warning] Unused function argument: `hasheous_on`

(ARG001)